### PR TITLE
Add support for Pyret

### PR DESCRIPTION
**Pull request by BNAndras**

### CODE REVIEW

**Comment by glennj** [comment]
Don't forget to update the README

**Comment by ErikSchierboom** [comment]
@BNAndras Could you rebase this on the latest `master` of this repo?

File: languages.json [hunk 1]
```diff
@@ -1148,6 +1148,12 @@
       "mime": ["text/x-python"],
       "extensions": ["py", "pyw"]
     },
+    "Pyret": {
+      "line_comment": ["#"],
+      "multi_line_comments": ["#|", "|#"],
```
Comment (glennj) [inline languages.json:1153]:
```suggestion
      "multi_line_comments": ["#|", "|#"],
      "nested": true,
```
As I discovered with 8th, you have to tell tokei that your language supports nested multiline comments.

**Review by glennj — CHANGES_REQUESTED** [review]

**Comment by BNAndras** [comment]
> @BNAndras Could you rebase this on the latest `master` of this repo?

This branch is also being used in a PR upstream. Will that be a problem for the other PR? I can change branches I think.

**Comment by ErikSchierboom** [comment]
@BNAndras I would probably just open a new PR for this repo with a different branch.

**Comment by BNAndras** [comment]
Okay, I opened #10 instead

**Pull request closed without merging**